### DISCUSSION TITLE
wechat group talking about for speed up bark  inference?

### DISCUSSION THREAD

**Discussion by EricKong1985** [suno-ai/bark#427] [General]
![微信图片_20230814145409](https://github.com/suno-ai/bark/assets/8405704/d7dfba20-4330-41f4-ad1f-80755c371f6d)

**Comment by minienglish1** [comment]
Could you provide a new qr code? It's expired.
能给新二维码吗？过期了.

**Comment by JonathanFly** [comment]
If somebody can translate any pro speed up tips, they'd be much appreciated. I can repay you in smooth talking Bark voices.

**Reply by minienglish1** [reply]
If you have access to some Chinese documentation, I can help translate if needed.  But realistically, if you have the modified code, run the accompanying Chinese through chatGPT/google translate, and it will be good enough for you to figure out what is going on.  That's usually what I do.

I'm not expecting anything amazing from the wechat group.  I'm more interested in the wechat group to learn more about how to use/access foreign based repositories such as github from China.
